spec/requests/notifications_spec.rb

```diff
@@ -44,7 +46,7 @@ def renders_comments_html(comment)
       it "renders the signup page" do
         get "/notifications"
 
-        expect(response.body).to include("Continue with")
+        expect(response.body).to include("By signing in")
```

Comment:
Is the page that renders on http://localhost:3000/notifications not the "login" page instead of the "signup page"?

<img width="1512" alt="Screenshot 2023-09-13 at 07 26 54" src="https://github.com/forem/forem/assets/2786819/f94e2108-8a96-4a55-b462-0e9abe110af3">

It also seems like both the signin(login) and signup page have that text "By signing in", is it possible to use more distinct text to signify a signin vs signup page?

Reply:
Same comment applies for the tests that are similar below.

Reply:
@Ridhwana If we open https://dev.to/notifications in incognito mode, we see that it shows "login" page similar to this PR's implementation. From UX perspective I think its correct because if someone is visiting the `/notifications` page then mostly likely they are a dev-signed-in-user, so showing login page makes sense. Let me know if I am missing something here.

Also, the text is slightly different on both the pages,
on login page it says -> `By signing in`
on signup page it says -> `By signing up`

This is the very small difference but atleast it is very close to the type of page it is. But I do agree that if we can improve this further it will be great, unfortunately, this is the most sensible thing I was able to find which defines the page too. 

The other option can be to use the title, i.e. `Join the` for signup pages and `Welcome back` for signin/login pages.  **Any thoughts on this?**

Reply:
https://github.com/forem/forem/pull/20065#discussion_r1324398182

Reply:
Regarding: 

> Also, the text is slightly different on both the pages,
on login page it says -> By signing in
on signup page it says -> By signing up

This sounds good! Thanks for clarifying.